includes/class-sensei-admin.php

```diff
@@ -1328,7 +1328,7 @@ public function lesson_order_screen() {
 						$count++;
 						$class = 'lesson';
 						if ( $count == 1 ) { $class .= ' first'; }
-						if ( $count == count( $lesson ) ) { $class .= ' last'; }
+						if ( $count == count( $lessons ) ) { $class .= ' last'; }
```

Comment:
`count( $lessons )` here counts the total number of lessons in the course. It looks like what we need for "Other Lessons" is the count of the number of just those other lessons. We should be seeing the `last` CSS class applied to the last "other lesson" as can be seen with the module lessons.